src/haiway/helpers/asynchrony.py

```
@@ -11,12 +11,10 @@
 
 
 @overload
-def asynchronous[**Args, Result]() -> (
-    Callable[
-        [Callable[Args, Result]],
-        Callable[Args, Coroutine[Any, Any, Result]],
-    ]
-): ...
+def asynchronous[**Args, Result]() -> Callable[
```

Pylint (3.3.7)

[error] Parsing failed: 'expected '(' (haiway.helpers.asynchrony, line 14)'

(E0001)